Casks/i/iexplorer.rb

```diff
@@ -10,11 +10,11 @@
   livecheck do
     url "https://macroplant.com/iexplorer/mac/v#{version.csv.first.major}/appcast"
```

Comment:
```suggestion
    url "https://macroplant.com/iexplorer/appcast"
```

This URL does not require specifying the major version and redirects to the latest appcast.